### DISCUSSION TITLE
infinity framework

### DISCUSSION THREAD

**Discussion by romainremyb** [infiniflow/ragflow#2026] [General]
Why isn't the infinity framework used in ragflow?

**Comment by JinHai-CN** [comment]
Infinity storage format may change before infinity 0.3.0 is released. Infinity will be used in RAGFlow, when 0.3.0 version of infinity is released.

**Reply by sbmandava** [reply]
Any plans ? to swap out Infinity for elastic search !!
[v0.5.0-dev1](https://github.com/infiniflow/infinity/releases/tag/v0.5.0-dev1)

**Reply by JinHai-CN** [reply]
We postponed the integration to next release of RAGFlow. Please check the status of PR #2894 .

**Comment by JinHai-CN** [comment]
PR #2894 is merged.
